lib/in_out/reader.py

```diff
@@ -2,39 +2,47 @@
 import sys
 sys.path.append('../lib')
 import config
+import numpy as np
+from datetime import date
+
+def linux_input(log_directory, channel_name, starting_date, starting_month,starting_year, ending_date, ending_month,ending_year):   
```

Comment:
function signature should be `def linux_input(log_directory, channel_name, starting_date, ending_date):`
where starting_date would be like '2012-01-20'
day, month and year should be calculated after splitting from starting_date